src/components/UriViewer/UriViewer.js

```diff
@@ -41,6 +43,13 @@ export const UriViewer = _.flow(
         })
         const metadata = await loadObject(googleProject, bucket, name)
         setMetadata(metadata)
+      } else if (isAzureUri(uri)) {
+        const loadObjectMetadata = withRequesterPaysHandler(onRequesterPaysError, () => {
+          return Ajax(signal).AzureStorage.blobMetadata(uri).getData()
+        })
```

Comment:
requester-pays is a GCP thing, isn't it? I suspect the `withRequesterPaysHandler` is extraneous here. Or at least, I suspect the `withRequesterPaysHandler` is specific to GCP.

Reply:
Yes, requester pays is specific to GCP so it is not needed here.

Reply:
@yuliadub can you remove the `withRequesterPaysHandler` wrapper here?

Reply:
done!